src/label_studio_sdk/prompts/versions/client.py

```diff
@@ -31,6 +31,7 @@ def create(
         provider: PromptVersionProvider,
         provider_model_id: str,
         parent_model: typing.Optional[int] = OMIT,
+        model_provider_connection: typing.Optional[int] = OMIT,
```

Comment:
it doesn't make any effect adding these fields manually - they will be overwritten.
Instead, do the following:
1. Find related api and schema here https://github.com/HumanSignal/label-studio-client-generator/blob/master/fern/openapi/resources/prompts.yaml
2. Change yaml file accordingly
3. Submit PR with `fb-dia-1422` branch

Then you'll see these changes.

Here is the full guide https://www.notion.so/humansignal/How-to-contribute-to-API-SDK-Documentation-7f6e6a80a5d84a9698bc0405211b8ca1

Reply:
made those changes